include/grpc/impl/codegen/port_platform.h

```diff
@@ -290,6 +290,12 @@
 #endif
 #endif /* GPR_NO_AUTODETECT_PLATFORM */
 
+#if defined(__has_include)
+#if __has_include(<atomic>)
+#define GRPC_HAS_CXX11_ATOMIC
```

Comment:
Are we going to support any C++ compiler without atomic? Does any such compiler exist while still being in the domain of C++11?